File: lista1_js/25.km_m.js
//25. Leia um número inteiro de metros, calcule e escreva quantos Km e quantos metros ele corresponde.

import { question } from "readline-sync"

//Entrada
const metros = Number(question('Informe um valor em metros: '))


//Processamento
const km = Math.floor(metros / 1000)
const m = metros % 1000

//Saída
console.log(`\n>>> O valor equivalente é de ${km.toFixed(0)}km e ${m.toFixed(1)}m.`)

File: lista1_py/25.km_m.py
#25. Leia um número inteiro de metros, calcule e escreva quantos Km e quantos metros ele corresponde.

#Entrada
metros = float(input('Informe um valor em metros: '))

#Processamento
km = metros // 1000
m = metros % 1000

#Saída
print(f'\n>>> O valor equivalente é de {km:.0f}km e {m:.1f}m.')